tide_webform.install

```diff
@@ -371,6 +371,8 @@ function tide_webform_update_8016() {
   $role_ids = ['editor', 'approver', 'site_admin', 'contributor'];
   foreach ($role_ids as $role_id) {
     $role = Role::load($role_id);
-    $role->grantPermission('filter.format.webform_default')->save();
+    if ($role) {
```

Comment:
contributor role does not exist for all projects, that is why this needs a check before assigning the permission, so that it does not throw null exception.